docroot/sites/library/settings/default.local.settings.php

```diff
@@ -158,3 +158,10 @@
 $settings['trusted_host_patterns'] = [
   '^.+$',
 ];
+
+// Automatically generated include for settings managed by ddev.
+$ddev_settings = dirname(__FILE__) . '/settings.ddev.php';
+if (getenv('IS_DDEV_PROJECT') == 'true' && is_readable($ddev_settings)) {
```

Comment:
You can probably just simplify this.
```suggestion
if (file_exists($ddev_settings)) {
```
Also maybe make it an array just in case we wish to add more files. That would make it more identical to other code like [this settings](https://github.com/SU-SWS/ace-sulgryphon/blob/2.x/docroot/sites/settings/global.settings.php#L55)

Reply:
I supposed they engineered it like that so there's no way it could run outside of a ddev environment. I suppose this is a non-issue because no one uses (or at least they shouldn't) use local.settings.php on any environment (except locally).

Reply:
yeah, I appreciate the extra checks, but it's not really needed. Even on local environments, you're probably the only one with the ddev file.

Reply:
@pookmish I just remembered that this snippet is from what Ddev puts in the settings file, which is why. (I forgot since I disabled this feature)

Anyway, I updated the foreach to be more scalable. Ready!